### PR TITLE
Disconnect user remote connections after logout

### PR DESCRIPTION
**Pull request by tedtoer**
https://github.com/rails/actioncable-examples/issues/7

Hi, I fixed the problem in issue above and did test
Sorry for possible defects - Its my first ruby-PR.

### CODE REVIEW

File: app/controllers/concerns/authentication.rb [hunk 1]
```diff
@@ -18,6 +18,10 @@ def authenticate_user(user_id)
     end
 
     def unauthenticate_user
+      ActionCable.server.remote_connections.where(
```
Comment (dhh) [inline app/controllers/concerns/authentication.rb]: Let's put this call on a single line.

**Comment by dhh** [comment]
Looking good!

Seeing this, I'd like to have a short-cut on remote connections for this default case:

`ActionCable.server.remote_connections.where(current_user: @current_user).disconnect`

Should have the following short-cut:

`ActionCable.server.disconnect(current_user: @current_user)`

Want to take take of that in another PR?

File: app/controllers/concerns/authentication.rb (hunk 1 above)
Comment (tedtoer) [inline app/controllers/concerns/authentication.rb]: I wanted to fit this in 80 characters limit, but probably in vain

**Comment by tedtoer** [comment]
@dhh thank you for your comments! Yes, better will new clean PR, if you haven't any remarks for test

**Pull request closed without merging**